opentelemetry-api/src/opentelemetry/trace/propagation/context/__init__.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Optional
+
+from opentelemetry.context import Context, current
+from opentelemetry.trace import INVALID_SPAN_CONTEXT, Span, SpanContext
+from opentelemetry.trace.propagation import ContextKeys
+
+
+def span_context_from_context(
+    context: Optional[Context] = None,
+) -> SpanContext:
+    span = span_from_context(context=context)
+    if span:
+        return span.get_context()
+    sc = current().value(ContextKeys.span_context_key(), context=context)  # type: ignore
```

Comment:
`current()` returns a snapshot of the current context but only a key/slot is used, shouldn't be a more straightforward way to get a single "slot" without creating a full snapshot?